packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -52,31 +52,8 @@ final RegExp _kBuildVariantRegex = RegExp('^BuildVariant: (?<$_kBuildVariantRege
 const String _kBuildVariantRegexGroupName = 'variant';
 const String _kBuildVariantTaskName = 'printBuildVariants';
 
-/// The regex to grab variant names from print${BuildVariant}ApplicationId gradle task
-///
-/// The task is defined in flutter/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy
-///
-/// The expected output from the task should be similar to:
-///
-/// ApplicationId: com.example.my_id
-final RegExp _kApplicationIdRegex = RegExp('^ApplicationId: (?<$_kApplicationIdRegexGroupName>.*)\$');
-const String _kApplicationIdRegexGroupName = 'applicationId';
-String _getPrintApplicationIdTaskFor(String buildVariant) {
-  return _taskForBuildVariant('print', buildVariant, 'ApplicationId');
-}
-
-/// The regex to grab app link domains from print${BuildVariant}AppLinkDomains gradle task
-///
-/// The task is defined in flutter/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy
-///
-/// The expected output from the task should be similar to:
-///
-/// Domain: domain.com
-/// Domain: another-domain.dev
-final RegExp _kAppLinkDomainsRegex = RegExp('^Domain: (?<$_kAppLinkDomainsGroupName>.*)\$');
-const String _kAppLinkDomainsGroupName = 'domain';
-String _getPrintAppLinkDomainsTaskFor(String buildVariant) {
-  return _taskForBuildVariant('print', buildVariant, 'AppLinkDomains');
+String _getDumpAppLinkSettingsTaskFor(String buildVariant) {
+  return _taskForBuildVariant('dump', buildVariant, 'AppLinkSettings');
```

Comment:
Can you make these constants that are referenced from here and in flutter.groovy?